### ISSUE TITLE
AWS Config

### ISSUE THREAD

**Issue by aaronwalker** [Optum/ChaoSlingr#43]
First of all.... great work on this project. 

I wanted to start a discussion on how ChaosSlingr fits into the overall echo system around continuous security and compliance and how it might be used in conjunction with AWS Config. There seems to be some overlap in some of the goals and functionality between ChaosSlingr and Config. It would be great to get an understanding on the overall direction for this project.

I've been thinking of working on something to help fill in some holes where Config falls short and think this project is a great start

**Comment by MichaelLiZhou** [comment]
there are a few similarities but i think config just keeps resource states of compliance. while this project is trying to facilitate changes to that state. 

i personally don't use config because it's hard to manage so I won't be able to help improve the feature set. i would like to hear what other conclusions you have drawn between the two.

**Comment by AARonSTeneo** [comment]
Do you guys wanna setup some time to chat? How about sometime next week? I can set something up just let me know when is good and a good contact email to send the meeting info to

**Comment by aaronwalker** [comment]
@AARonSTeneo yeah happy to catch up for a chat.  my email is aaronpwalker at gmail dot com
